xapian-core/tests/termgentest.cc

```diff
@@ -152,6 +161,19 @@ static const test test_simple[] = {
     { "all_z",
 	  "Only stemmed words!", "Zonli[1] Zstem[2] Zword[3]" },
 
+    // Test set_stopper_strategy():
+    { "stop=en,none,stop_none",
+    	"The stop words.", "stop[2] the[1] words[3]" },
+
+    { "stop_all",
+    	"The stop words.", "stop[1] words[2]" },
+
+    { "stop_stemmed",
+    	"The stop words.", "stop[2] the[1] words[3]" },
+
+    { "stem=en,all,stop_stemmed",
+    	"The stemmed words.", "stem[2] the[1] word[3]" },
+
```

Comment:
Given the stopper persists, shouldn't we end this set with the `stop_none` test, so it's equivalent to not having a stopper configured?